Add converter from bitstream to openOCD script

kowalewskijan opened the pull request.

Commit:
Add converter from bitstream to OpenOCD script

Signed-off-by: Jan Kowalewski <jkowalewski@antmicro.com>

mkurc-ant commented:
@kowalewskijan There is also a script for configuring the IOMUX in SymbiFlow. It uses JSON as well but with a different data format. Could make this one to accept that data format?

https://github.com/QuickLogic-Corp/symbiflow-arch-defs/blob/quicklogic-upstream-rebase/quicklogic/common/utils/eos_s3_iomux_config.py

mkurc-ant reviewed (approved):
One comment about the JSON format for IOMUX config. Otherwise LGTM.

Commit:
Add OpenOCD script generating manual to README

Signed-off-by: Jan Kowalewski <jkowalewski@antmicro.com>

kowalewskijan commented:
I removed parsing JSONs from this PR since we have a tool capable of this already. I created a separate PR for generating openocd script which handles IOMUX [here](https://github.com/antmicro/symbiflow-arch-defs/pull/18). I also added README entry for this script.
